plugins/uefi/uefi.quirk

```diff
@@ -0,0 +1,11 @@
+# Lenovo ThinkStation P920/P720
+[DeviceInstanceId=cc850e7a-6713-4fe3-8144-321a9e485ae0]
+VersionFormat = intel-me
+
+# Lenovo ThinkStation P520/P520c
+[DeviceInstanceId=e3af51e5-5afb-4b48-b5cb-13bf26becaab]
+VersionFormat = intel-me
+
+# Lenovo ThinkStation P330/P330 Tiny & ThinkCentre M920X/M920q
+[DeviceInstanceId=5b92717b-2cad-4a96-a13b-9d65781df8bf]
```

Comment:
> Is this the case for a whole class of machines

I think so, I think it's tied more to the generation of ME more than the machine itself. Those are more examples of shipping systems.

Reply:
Is this the case of an FMP update then that's not primary firmware on the system? Can that be included in the quirk somehow? Eg if it's Lenovo and more than one ESRT entry?

Reply:
> Is this the case of an FMP update

I don't think it's FMP, I only have 1 `system` and 2 `device` entries in my P50 ESRT.

Reply:
Is it one of the device entries?

Reply:
I booted up a P720 I have under my desk. It has two ESRT entries, one device, one system. The device ESRT entry matches `cc850e7a-6713-4fe3-8144-321a9e485ae0`

Reply:
Can you confirm with Lenovo it always matches the device entry when it's around?  If so it seems like that should be part of the quirk too?

Reply:
Before I realized that the DeviceId was actually a GUID from the ESRT I was thinking it was a hardware ID and this would help to narrow it down to the proper UEFI device.  It might be clearer to also use Guid here too for that reason.

Reply:
> I was thinking it was a hardware ID

Aha, gotcha. In fwupd a 1`HardwareInstanceId` is indeed a GUID, but only because we make it one. I agree using `Guid=` makes it more obvious when not using the `VID=foo&PID=bar` format.